src/features/surveys/components/surveyForm/SurveySuccess.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box } from '@mui/material';
+import { FC } from 'react';
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import { ZetkinSurveyExtended } from 'utils/types/zetkin';
+
+export type SurveySuccessProps = {
+  survey: ZetkinSurveyExtended;
+};
+
+const SurveySuccess: FC<SurveySuccessProps> = ({ survey }) => {
+  return (
+    <Box>
+      <h1>
```

Comment:
For this we would normally use a `<Typography variant="h1">` (IIRC).

Reply:
Fixed in daaba32837bcb644a4a93be5f1b408a68781361e!

Reply:
This was referring to the `h1`. Changing the `p` was also good (and I didn't leave a comment there because I had already commented on another `p`) but I think the `h1` should also be changed.

Reply:
lmao this seems _very_ obvious now, fixed for realsies this time in e6439eb2be5d74366d02ddfe8a001dbff9e7200a!